docs/index.md

```diff
@@ -33,10 +33,11 @@ We have also taken strategic product decisions which has shaped Ultra offering f
 
 # Why Ultra
```

Comment:
Focus more on why ultra for partners here and then bring user angle? End-users won't be reading this document. The partner devs or biz folks reading this should get an idea about what's in it for them